src/bin/rustfmt.rs

```diff
@@ -136,6 +136,16 @@ fn lookup_project_file(dir: &Path) -> FmtResult<Option<PathBuf>> {
     }
 }
 
+fn resolve_config_file_path(file_path: &Path) -> FmtResult<(Config, Option<PathBuf>)> {
```

Comment:
A better name for this function might be `open_config_file` or something similar